packages/tenant-process/src/routers/TenantRouter.ts

```diff
@@ -323,7 +324,26 @@ const tenantsRouter = (
     .post(
       "/tenants/:tenantId/attributes/verified",
       authorizationMiddleware([ADMIN_ROLE]),
-      async (_req, res) => res.status(501).send()
+      async (req, res) => {
+        try {
+          const { tenantId, offset, limit } = req.query;
+          const tenant = await tenantService.verifyVerifiedAttribute({
+            tenantId: unsafeBrandId(tenantId),
+            tenantAttributeSeed: req.body,
+            authData: req.ctx.authData,
+            limit,
+            offset,
```

Comment:
```suggestion
          const tenant = await tenantService.verifyVerifiedAttribute({
            tenantId: unsafeBrandId(req.path),
            tenantAttributeSeed: req.body,
            authData: req.ctx.authData,
```
This endpoint only has `tenantId` in its path, it doesn't have `limit` and `offset`.